docs/user-guide/api-mediation/discovery-service-configuration.md

```diff
@@ -0,0 +1,68 @@
+# Discovery Service configuration parameters
+
+## Zowe runtime configuration parameters
+
+As an application developer who wants to run Zowe, set the following parameters during the Zowe runtime configuration by modifying the `<Zowe install directory>/components/api-mediation/bin/start.sh` file:
+
+* **[API ML configuration](#api-ml-configuration)**
+* **[Eureka configuration](#eureka-configuration)**
+
+## API ML configuration
+
+* **apiml.discovery.userid**
+
+    The Discovery service in HTTP mode protects it's endpoints with basic authentication instead of client certificate. This parameter  specifies the userid. The default value is `eureka`. 
+
+* **apiml.discovery.password**
+
+    This parameter specifies the password for the basic authentication used by the Discovery Service in HTTP mode. The default value is `password`.
+    
+* **apiml.discovery.allPeersUrls**
+
+    This parameter contains the list of URLs of the Discovery Service in case of multiple instances. 
```

Comment:
I would mention that we expect the services to be on different host and that there should be comma between them.